.github/workflows/test.yml

```
@@ -38,9 +38,9 @@ jobs:
       - name: Install 19
         if: ${{ matrix.react == '19' }}
         run: |
-          yarn set resolution "@types/react@npm:*" "npm:types-react@rc"
-          yarn set resolution "@types/react-dom@npm:*" "npm:types-react-dom@rc"
-          yarn add react@rc react-dom@rc
+          yarn set resolution "@types/react@npm:*" "npm:types-react@19"
+          yarn set resolution "@types/react-dom@npm:*" "npm:types-react-dom@19"
```

```suggestion
          yarn add "@types/react@19" "@types/react-dom@19" --dev
```
React 19 types are now stable